test/user.test.js

```diff
@@ -223,6 +265,23 @@ describe('User', function() {
       });
     });
 
+    it('Login a user by providing email credentials (email case-sensitivity off)', function(done) {
+      User.settings.caseSensitiveEmail = false;
+      User.login(validCaseInsensitiveEmailCredentials, function(err, accessToken) {
+        assert(accessToken.userId);
+        assert(accessToken.id);
+        assert.equal(accessToken.id.length, 64);
+        done();
+      });
+    });
+
+    it('Try to login with invalid email case', function(done) {
+      User.login(validCaseInsensitiveEmailCredentials, function(err, accessToken) {
```

Comment:
Good catch, @superkhau. I think the name `validCaseInsensitiveEmailCredentials` is confusing. The variable contains credentials that are invalid by default but valid when case-insensitive check is made.